compiler/control/OMROptions.hpp

```diff
@@ -1072,6 +1072,7 @@ enum TR_VerboseFlags
    TR_VerboseIProfilerPersistence,
    TR_VerboseCheckpointRestore,
    TR_VerboseCheckpointRestoreDetails,
+   TR_VerboseFootprint,
```

Comment:
Thanks. I will think about it. **METHOD STATS** refers more to the format of the output. It can be any `key=value` pairs. It's just this PR happened to print footprint related statistics. This statistics will be need for a footprint reduction work that I am planning to check in. That's why the name of the option is Footprint.

Reply:
I think you are right. I simply removed that new option for now.